scripts/autograph_travis_test_config.yaml

```diff
@@ -131,7 +131,7 @@ signers:
         path: "mozilla-recommendation.json"
```

Comment:
Not part of this change, but is the right path or just used for tests? Autograph has `recommendation.json` in their docs https://github.com/mozilla-services/autograph/blob/515ce3638d1b8a8cab1543baf6f33f2223c12831/signer/xpi/README.rst#id1

Reply:
looks like their readme is wrong
https://github.com/mozilla-services/autograph/blob/master/autograph.yaml#L987

(and yeah, it's just for tests and local docker dev)